pkg/hive/resources.go

```diff
@@ -193,6 +193,11 @@ func adoptedClusterDeployment(namespace, clusterName, clusterID, infraID, locati
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ClusterDeploymentName,
 			Namespace: namespace,
+			Annotations: map[string]string{
+				// https://github.com/openshift/hive/pull/2501
+				// Disable hibernation controller
```

Comment:
Tiniest nit, but it's not clear looking at the code or the linked PR why we're disabling it.
```suggestion
				// Disable hibernation controller as it is not used as part of ARO's Hive implementation 
```